### ISSUE TITLE
[BUG] UITextAttributes - hasOrderedList looking into hasLink

### ISSUE THREAD

**Issue by leite-paulohf** [Infomaniak/swift-rich-html-editor#10]
# TextAttributes.swift - Line 50
```
public final class TextAttributes: ObservableObject {

    (...)

    func update(from uiTextAttributes: UITextAttributes) {
        (...)
        
        hasOrderedList = uiTextAttributes.hasLink
        
        (...)
    }
}

```

**Comment by valentinperignon** [comment]
Hello @leite-paulohf,
Nice catch! Thank you for your help. This is now fix by #11

**Comment by valentinperignon** [comment]
The fix is available in the version 1.4.1.
